### DISCUSSION TITLE
Question - Get event by date

### DISCUSSION THREAD

**Discussion by sabatale** [nhn/tui.calendar#1304] [Q&A] [answered]
## Version
Latest.

## Test Environment
Edge latest.

## Current Behavior
Is it possible to get an event by date? I know `getElement()` and `getEvent()` exist but they both need the `event.id` parameter as input. We're trying to detect potential overlapping events through API so we would ideally do something like `cal.getEventsByDate("2022-10-01", 1)` to verify if an event already exists before allowing the user to create a new event.

Would you happen to have a workaround to list all events on a given calendar for a particular date?

We didn't find such a method in the docs.

**Comment by adhrinae** [comment, marked as answer]
We don't have that feature because you can already fully control your event data.
Save data wherever you want and create a function to filter them by date.
